lib/active_record/connection_adapters/postgis/setup.rb

```diff
@@ -2,7 +2,16 @@ module ActiveRecord  # :nodoc:
   module ConnectionAdapters  # :nodoc:
     module PostGIS  # :nodoc:
       def self.initial_setup
-        ::ActiveRecord::SchemaDumper.ignore_tables |= %w(geometry_columns spatial_ref_sys layer topology)
+        ::ActiveRecord::SchemaDumper.ignore_tables |= %w(
+          geometry_columns
+          spatial_ref_sys
+          layer
+          topology
+          geography_columns
+          geometry_columns
```

Comment:
`geometry_columns` is in the list twice. Can you de-dupe & alphabetize these? Thanks!